packages/cozy-client/src/store/index.js

```diff
@@ -49,33 +49,65 @@ export class StoreProxy {
 
 const initialState = { documents: {}, queries: {} }
 
-const combinedReducer = (state = initialState, action) => {
+const combinedReducer = performanceApi => (state = initialState, action) => {
+  const markName = performanceApi.mark('combinedReducer')
+
   if (action.type == RESET_ACTION_TYPE) {
+    performanceApi.measure({
+      markName: markName,
+      measureName: `${markName} reset`,
+      category: 'CozyClientStore'
+    })
```

Comment:
The idea was to have all execution path measured even if some are instant so the output is more consistent.

For example, if there 4 methods calls that should call the reducer, then I prefer to have 4 performance entry even if some of them are very small, instead of having only some of them and wondering why some calls are not logged (is that due to an exception? is the reducer called? etc).

But for this particular example, the `reset` is rare enough so the example I explained is not relevant. So have no objection to remove it.

Reply:
Ok, fair enough for the consistency, ok to keep it like this